src/mbgl/map/tile_worker.cpp

```diff
@@ -91,11 +90,8 @@ void TileWorker::redoPlacement(
     // Reset the collision tile so we have a clean slate; we're placing all features anyway.
     collisionTile = std::make_unique<CollisionTile>(config);
 
-    for (auto i = layers.rbegin(); i != layers.rend(); i++) {
```

Comment:
@kkaefer Do you see any issues here with iterating `buckets` directly, rather than `layers`?